gradle.properties-example

```diff
@@ -31,6 +31,10 @@ wp.sentry.dsn=https://00000000000000000000000000000000@sentry.io/00000000
 # Needed to use the Google Maps component aka the PlacePicker (Post Settings -> Location)
 wp.res.com.google.android.geo.api.key = geo-api-key
 
+# Uncomment this and update the path to use a local copy of WordPressMocks
+# Note the path should be relative to the project root
```

Comment:
These instructions were a little confusing to me because at first I thought the path should point from the `WordPress-Android` project root to the `WordPressMocks` project root. After trying it out I realized it need to point to the `WordPressMocks` directory within that project's root directory. The example alludes to this but I'm not sure if there's a clear way to explain that here.

Reply:
Thats a fair point. I'll update the instructions.

Reply:
Okay, its updated now with clearer instructions but also to remove the need to point to the `WordPressMocks` subdirectory.

Reply:
Looks great! Looks like the old instructions are still in this file (above) but the new instructions (below) and change are awesome. :)